param/parameterized.py

```diff
@@ -209,6 +209,25 @@ def _f(*args, **kwargs):
     return _f
 
 
+def disable_instance_params(cls):
+    """
+    Disables instance parameters on the class
+    """
+    cls._disable_instance__params = True
```

Comment:
What's with the __?

Reply:
Was following the weird pattern we use to cache parameters on classes, i.e.:

```
'_Parameterized__params'
```